### PR TITLE
WFLY-4928 Feature pack modules are built in the wrong order

### PR DESCRIPTION
**Pull request by Sanne**
https://issues.jboss.org/browse/WFLY-4928

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -308,6 +307,7 @@
         <module>web-common</module>
         <module>servlet-dist</module>
         <module>servlet-feature-pack</module>
+        <module>feature-pack</module>
```
Comment (ctomc) [inline pom.xml]: that shouldn't matter at all. as long as there are proper dependencies in place, maven will figure out the proper order.

Comment (Sanne) [inline pom.xml]: I double-checked, and it seems you're right: my local build is working fine now even without this patch. I'm wondering though if something is now locally cached.

You have a better theory to explain the build error I had ?

  > [ERROR] Failed to execute goal on project wildfly-feature-pack: Could not resolve dependencies for project org.wildfly:wildfly-feature-pack:pom:10.0.0.Alpha6-SNAPSHOT: Failure to find org.wildfly:wildfly-servlet-feature-pack:zip:10.0.0.Alpha6-SNAPSHOT in http://repository.jboss.org/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of jboss-public-repository-group has elapsed or updates are forced -> [Help 1]

Comment (ctomc) [inline pom.xml]: how do you build it? just mvn clean install or any special params?

Comment (Sanne) [inline pom.xml]: As the `README` suggests:

    ./build.sh test

Comment (stuartwdouglas) [inline pom.xml]: I think you need to do ./build.sh install to install the servlet feature pack into the local repository.

**Comment by Sanne** [comment]
sorry for the delay, I've been away.

Thanks, that makes sense I probably built it the wrong way, and can't reproduce it anymore.

**Pull request closed without merging**